src/components/App.js

```diff
@@ -18,6 +18,14 @@ const App = () => {
     setInput('')
   }
 
+  const updateToDo = (id, texto) => {
```

Comment:
`todo` en inglés se escribe todo en minúsculas, si bien son dos palabras separadas, se trata como si fuera una sola, por lo que por consistencia es mejor mantenerla así.

Por otro lado, `texto` está en español.

```suggestion
  const updateTodo = (id, text) => {
```

Reply:
Pablo, pregunta: cuando arreglo esto y lo commiteo, acá te aviso? Te dejo comment o resuelvo la conversación?

Reply:
Los cambios los veo recién cuando pushéas, cuando lo hagas podés cerrarlo o dejar comentario, como prefieras, no hay realmente un estándar al respecto, total en ambos casos lo veo igual y lo puedo modificar llegada la situación.